### PR TITLE
fix(extends): resolve absolute paths

### PR DESCRIPTION
**Pull request by andykog**
As asked in #39, this PR demonstrates a failing tests for absolute path

### CODE REVIEW

File: tests/specs/parse-tsconfig/extends/extends.spec.ts [hunk 1]
```diff
@@ -81,6 +83,31 @@ export default testSuite(({ describe }) => {
...
+						compilerOptions: {
+							strict: true,
+							jsx: 'react',
+						},
+					}),
+					'tsconfig.json': tsconfigJson({
+						extends: absolutePath,
+					}),
+					'file.ts': '',
+				});
```
Comment (privatenumber) [inline tests/specs/parse-tsconfig/extends/extends.spec.ts:100]: You can do `fixture.writeFile()` to create the `tsconfig.json` file that references `fixture.path` for the absolute path.

Comment (andykog) [reply tests/specs/parse-tsconfig/extends/extends.spec.ts:95]: Good idea, fixed

**Comment by andykog** [comment]
@privatenumber made the test pass

**Pull request merged**